readme/Readme.scalatex

```diff
@@ -322,6 +335,89 @@
                     To explicitly isolate a cut to one branch of a parser, place that branch within @hl.scala{NoCut}.  Cuts within that branch will prevent backtracking inside that branch, but if that branch fails alternate branches will be tried as normal.
 
                 @hl.ref(tests/"ExampleTests.scala", Seq("'composenocut", ""))
+        @sect{Unapply}
+            @p
+                @hl.scala{Parser} class has (as many other classes in Scala) a convenient method @hl.scala{unapply}
+                which allows to do pattern matching using parsers (like regular expressions) in your code.
+
+                @hl.ref(tests/"ExampleTests.scala", "val capture1", "val captureOpt")
+
+
+    @sect{Writing Byte Parsers}
+        @p
+            In the same way as the @hl.scala{String} parsers we can build @hl.scala{Array[Byte]} or simply @hl.scala{Byte} parsers.
+        @p
+            The first example will be also about parsing a single byte sequence
+        @hl.ref(bytetests/"ByteTests.scala", Seq("'sequence", ""))
+
+        @p
+            As you can see it has the very same structure as previous @hl.scala{String} parsers,
+            but with several additional features.
+            The first is new word @hl.scala{BS} or @hl.scala{ByteSeq} that merely means alias for @hl.scala{Array[Byte]},
+            the second is new function for convenient conversion from @hl.scala{String} to @hl.scala{Array[Byte]} - @hl.scala{strToBytes}.
+        @p
+            Of course, you aren't restricted to use only them, you are free to create Array of Byte by any method or function,
+            but, naturally, it's recommended to follow the one uniform style and @hl.scala{BS} with @hl.scala{strToBytes} is the good example of it.
+
+        @sect{New Parsers}
+            @p
+                Byte Parsers doesn't bring anything new to the basic set of parsers.
+                @hl.scala{Option}, @hl.scala{Repeat}, @hl.scala{Capture} and ect. are still here with the same logic,
```

Comment:
These should all be `@sect.ref`s to link to the relevant sections of the string-parser docs